index.js

```diff
@@ -42,21 +80,57 @@ const queryDns = (version, options) => {
 
 	const socketQuery = promisify(socket.query.bind(socket));
 
-	// eslint-disable-next-line promise/prefer-await-to-then
-	const promise = socketQuery({questions: [data.dnsQuestion]}, 53, data.dnsServer).then(({answers}) => {
-		socket.destroy();
+	const promise = (async () => {
+		let internalPromise = Promise.resolve();
+
+		data.dnsServers.forEach(dnsServerInfo => {
```

Comment:
I prefer `for-of` loops instead of forEach.

Reply:
tried to use for-in but linter had problems with that and tried to do something that did not need ignoring it ... can change it back if wanted